server/src/main/java/org/spine3/server/reflect/HandlerMethod.java

```diff
@@ -41,9 +45,14 @@
  *
  * @author Mikhail Melnik
  * @author Alexander Yevsyukov
- * @param <C> the type of the message context or {@code Void} if context is not used
+ * @param <C> the type of the message context or {@link Empty} if context is not used
```

Comment:
Please change the wording `if context` into `if a context parameter is never used`. It would stress on the fact that the parameter may or may not be used, but there are cases where there is only one parameter only.